iceoryx_posh/include/iceoryx_posh/internal/popo/building_blocks/chunk_queue_popper.inl

```diff
@@ -109,22 +109,14 @@ inline uint64_t ChunkQueuePopper<ChunkQueueDataType>::getMaximumCapacity() const
 template <typename ChunkQueueDataType>
 inline void ChunkQueuePopper<ChunkQueueDataType>::clear() noexcept
 {
-    do
+    while (auto maybeChunkTuple = getMembers()->m_queue.pop())
```

Comment:
nitpicking, but this is not a maybeChunkTuple. The loop is not entered if it's a nullopt

Reply:
Still, I would suggest to use the `maybe*` everywhere where an `optional` or `expected` is returned. This triggers the reminder in my brain "Hey that's an optional/expected!"

Reply:
Valid point. In the loop body it is a chunk tuple. But when we do the assignment it is maybe one ;-)